examples/tests/sdk_tests/topic/no_blocking.test.w

```diff
@@ -18,8 +18,8 @@ test "topic subscribers are invoked without blocking" {
   // since the subscriber sleeps for 3 seconds, the counter should still be 0
   assert(counter.peek() == 0);
 
-  util.sleep(6s);
-
   // after a while, the counter should be 1
+  util.waitUntil(inflight () => { return counter.peek() == 1; }, interval: 1s, timeout: 30s);
+
   assert(counter.peek() == 1);
```

Comment:
We technically don't need the assert anymore, but leaving it in is good for clarity I guess

Reply:
Hmm, yeah I guess I wasn't sure from reading the docstring

> Run a predicate repeatedly, waiting until it returns true or until the timeout elapses

Is it supposed to throw if it times out?

Reply:
It should, but just now looking at the code it does not lol